### ISSUE TITLE
[Accessibility] Buttons, Images and Links are not accessibile

### ISSUE THREAD

**Issue by playstore777** [alexdeploy/developer-portfolio-v2#31]
There are currently two types of accessibility issues in Hero, About, Projects, and Contact page:

1. Interactive elements - do not affect the logic or aesthetics.
2. Color contrast - does not affect the logic, however might affect the looks.

**Interactive elements:**
- Buttons do not have accessible names.
- Images do not have an `alt` attribute.
- Links do not have a name for the screen reader.
- Forms elements such as checkboxes do not have a connection with labels.

What to do?
For Images, you can just add an `alt` attribute, if there is no need for an `alt` attribute then simply leave it empty without value, this will tell the screen reader to skip that image.

For Buttons and Links, you can use the `aria-label` or `alt` attribute if the only child is an `img` tag.

For Forms, just make sure to give the `for` attribute and `id` attribute the same value for the `label` tag and `input` tag respectively. In some places either one is missing, add them there.

**Color contrast:**
All the underlined (red color) text color should be updated to follow the accessibility rules.

![Image](https://github.com/user-attachments/assets/3526c9d9-060c-46fd-ad5e-4d7c62118aef)

**And the most important one, as this one also affects the SEO.**

`lang` attribute in the `html` element.

Since we do not have a `.html` file in NUXT, we have to update the `nuxt.config` file.
You can add the following, 
```
htmlAttrs: {
        lang: "en",
      },
```

If all are satisfied then the Accessibility will score 95+ on the lighthouse (chrome devtools) from 68.

![Image](https://github.com/user-attachments/assets/a4bd4c09-0f6d-4a9b-b39f-6261a724b95b)

**Comment by alexdeploy** [comment]
Thank you for your detailed report and for your contribution! 🎉

We've reviewed and merged the changes addressing the accessibility issues. The improvements in interactive elements and color contrast will help make the site more inclusive and user-friendly.

If you notice anything else that could be improved, feel free to share your thoughts. Your input is greatly appreciated! 🙌

**Comment by playstore777** [comment]
Glad you found it helpful! 😊 Absolutely, feel free to reach out if there's anything else I can improve. I'm on a mission to make the online world more inclusive, so if an open-source platform needs accessibility work, I'd love to help!
